reference/lib/ReferenceOrderCombiner.sol

```diff
@@ -481,8 +483,11 @@ contract ReferenceOrderCombiner is
 
             // If offerer and recipient on the execution are the same...
             if (execution.item.recipient == execution.offerer) {
-                // increment total filtered executions.
-                totalFilteredExecutions += 1;
+                unchecked {
+                    // increment total filtered executions.
+                    // totalFilteredExecutions starts at 0, infeasible to increment > 2^256.
```

Comment:
One tiny nit — let's ensure that comments do not cause the 80-character line limit to be exceeded

Reply:
Fixed :)